components/raftstore-v2/src/lib.rs

```diff
@@ -27,7 +27,8 @@
 #![feature(box_into_inner)]
 #![feature(assert_matches)]
 #![feature(option_get_or_insert_default)]
-#[allow(clippy::needless_pass_by_ref_mut)]
+#![allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
You can add it to Makefile.

Reply:
But I only want to allow this for some components temporarily. Some are not easy to fix, so we can fix them in following PRs.